pkg/config/initoptions_test.go

```diff
@@ -31,8 +68,11 @@ func TestComplete(t *testing.T) {
 			args:    []string{"foo"},
 			isError: true,
 		},
+		"More than one namespace should fail": {
```

Comment:
We should have tests here that cover resources where namespace has not been set.

Reply:
Done.